Automattic/jetpack
# Boost: Migrate Tips to react

## Proposed changes:

* Update Tips to react.

### Other information:


## Jetpack product discussion



pc9hqz-2jy-p2

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup Boost and make sure the Tips section is working as expected.

<img width="865" alt="CleanShot 2023-12-18 at 20 08 46@2x" src="https://github.com/Automattic/jetpack/assets/11799079/c2dfaf90-f3d2-4f57-bf15-4dee76e21db8">
